fee_allocator/accounting/distribution.py

```diff
@@ -77,8 +64,10 @@ def re_distribute_incentives(
         aura_vebal_share: Decimal
 ) -> Dict[str, Dict]:
     """
-    If some pools received < min_vote_incentive_amount all incentives from that pool
-        should be distributed to pools that received > min_vote_incentive_amount by weight
+    Redistribute all incentives away from pools that are < min_vote_incentive amount
```

Comment:
This function was changed quite a bit to operate in multiple stages.  Still needs refactoring but this leads in that direction.

Reply:
This makes total sense. First define total amounts, then distribute